src/test/resources/application-internal-api.yml

```diff
@@ -0,0 +1,11 @@
+fint:
+  flyt:
+    resource-server:
+      security:
+        api:
+          internal:
+            enabled: true
+            permit-all: false
+            authorized-org-ids:
+              - example.no
```

Comment:
Burde disse vært kommentert ut? Kan dummy-verdier utnyttes om noen glemmer å overskrive etc?

Reply:
Disse brukes kun om man kjører applikasjonen med aktiv profil "internal-api" fra test-området, så liten risiko for at det havner i prod.